programs/programs/il/medicaid/emergency/tests.py

```
@@ -0,0 +1,182 @@
+            white_label=self.il_white_label,
+        )
+        member = HouseholdMember.objects.create(
+            screen=screen,
+            relationship="headOfHousehold",
+            age=35,
+        )
+        Insurance.objects.create(household_member=member, none=True)
+
+        # Mock the medicaid_eligible helper
+        data = {"medicaid_eligible": True}
+        calc = self.create_calculator(screen, data)
```

Ruff (0.14.14)

Local variable `calc` is assigned to but never used

Remove assignment to unused variable `calc`

(F841)